development-docs/DEV_GUIDE.md

```diff
@@ -91,6 +91,10 @@ The `make` build is using GNU versions of `find` and `sed` utilities and is not
 
 The build requires `bash` version 4+ which is not shipped Mac OS but can be installed via homebrew. You can run `brew install bash` to install a compatible version of `bash`. If you wish to change the default shell to the updated bash run `sudo bash -c 'echo /usr/local/bin/bash >> /etc/shells'` and `chsh -s /usr/local/bin/bash`
 
+The `mvn` tool might install the latest version of openJDK during the brew install. For builds on Mac OS to succeed, openJDK version 11 needs to be installed. This can be done by running `brew install openjdk@11`. For mavern to read the new Java version, you will need to edit the `~/.mavernrc` file and paste the following line `export JAVA_HOME=/Library/Java/JavaVirtualMachines/openjdk-11.jdk/Contents/Home`.
```

Comment:
maven I guess?

```suggestion
The `mvn` tool might install the latest version of openJDK during the brew install. For builds on Mac OS to succeed, openJDK version 11 needs to be installed. This can be done by running `brew install openjdk@11`. For maven to read the new Java version, you will need to edit the `~/.mavenrc` file and paste the following line `export JAVA_HOME=/Library/Java/JavaVirtualMachines/openjdk-11.jdk/Contents/Home`.
```